### DISCUSSION TITLE
Federal moving away from CentOS

### DISCUSSION THREAD

**Discussion by jdpadro** [Security-Onion-Solutions/securityonion#4524] [Unsupported Versions] [answered]
I attended a meeting today in which the topic was to move completely move away from CentOS. This is still being discussed, but thought you should be made aware of it.

**Comment by TOoSmOotH** [comment, marked as answer]
@jdpadro Please see https://blog.securityonion.net/2020/12/security-onion-2-and-future-of-centos.html

CentOS is supported until June of 2024 so there is no rush at the moment for us to migrate to Rocky although that is our current plan. I think this is something we will most likely tackle in 2022 since making the move to Rocky will require the use of Podman over docker.
